packages/fast-usdc/src/exos/status-manager.js

```diff
@@ -86,25 +70,71 @@ export const prepareStatusManager = (
     log = makeTracer('Advancer', true),
   } = /** @type {StatusManagerPowers} */ ({}),
 ) => {
-  /** @type {MapStore<PendingTxKey, PendingTx[]>} */
+  /**
+   * Keyed by a tuple of the Noble Forwarding Account and amount.
+   * @type {MapStore<PendingTxKey, PendingTx[]>}
+   */
   const pendingTxs = zone.mapStore('PendingTxs', {
     keyShape: M.string(),
     valueShape: M.arrayOf(PendingTxShape),
   });
 
-  /** @type {SetStore<SeenTxKey>} */
+  /**
+   * Transactions seen *ever* by the contract.
+   *
+   * Note that these are stored in IAVL and grow monotonically. At some point in the future we may want to prune them.
```

Comment:
I wonder what "monotonic" is intended to communicate. Monotonic is a relationship between 2 variables. Is the dependent variable here time? A constant function is non-decreasing and hence monotonic.

```suggestion
   * Note that like all durable stores, this SetStore is stored in IAVL. It grows without bound (though the amount of growth per incoming message to the contract is bounded). At some point in the future we may want to prune it.
```

Reply:
>  the dependent variable here time?

Yes, or blockheight, what have you.

> A constant function is non-decreasing and hence monotonic.

Sure, it could stay constant if it's never used.

The suggestion is more clear, I'll take it.